docs/.vuepress/config.js

```diff
@@ -12,9 +12,16 @@ module.exports = {
             rel: 'stylesheet',
             type: 'text/css',
             href: 'https://fonts.googleapis.com/css?family=Nunito:200,200i,300,300i,400,400i,600,600i,800,800i,900,900i'
-        },]
+        },],
+        ['link', {
+            rel: 'stylesheet',
+            type: 'text/css',
+            href: 'https://fonts.googleapis.com/css?family=Miriam+Libre:400,700'
+        },],
     ],
+    theme: 'default-prefers-color-scheme',
     themeConfig: {
+        defaultTheme: 'light',
```

Comment:
I would be fine with defaulting to dark 😄

Reply:
I'll let you guys be the one to change that! I think light is a sensible default for those in well-lit environments who might not have access to a browser / OS with `prefers-color-scheme` support.